docs/en/interfaces/formats/Png.md

```diff
@@ -0,0 +1,53 @@
+---
+alias: []
+description: 'Documentation for the PNG image format'
+input_format: false
+keywords: ['Png']
+output_format: true
+slug: /interfaces/formats/Png
+title: 'Png'
+---
+
+
+| Input | Output | Alias |
+|-------|--------|-------|
+| ✗     | ✔      |   ✗   |
+
+	
+## Description {#description}
+
+Export query results directly as PNG images for visualization purposes. This format converts numeric data into pixel values, supporting `RGBA`, `RGB`, `grayscale`, and `binary` pixel formats with `8` or `16-bit` color depth and compression level control
+
+## Example Usage {#example-usage}
+
+```sql
+SELECT 
+    r, g, b, a 
```

Comment:
We also need an option for filling pixels by their coordinates: x, y, r, g, b, a:
https://github.com/ClickHouse/ClickHouse/issues/57834